Sources/Web3Core/KeystoreManager/KeystoreParams.swift

```diff
@@ -47,21 +75,7 @@ public struct KeystoreParamsBIP32: AbstractKeystoreParams {
     public var version: Int
     public var isHDWallet: Bool
 
-    @available(*, deprecated, message: "Please use pathAddressPairs instead")
-    var pathToAddress: [String: String] {
-        get {
-            return self.pathAddressPairs.reduce(into: [String: String]()) {
-                $0[$1.path] = $1.address
-            }
-        }
-        set {
-            for pair in newValue {
-                self.pathAddressPairs.append(PathAddressPair(path: pair.0, address: pair.1))
-            }
-        }
-    }
-
-    var pathAddressPairs: [PathAddressPair]
+    public var pathAddressPairs: [PathAddressPair]
```

Comment:
Let's make it `public private(set) var` just for safety.

Reply:
Marked as `public internal(set) var` instead because using private is not an option as it will break the uses of this field across the library.